framework/Models/ExternalModel.py

```diff
@@ -164,34 +180,51 @@ def localInputAndChecks(self,xmlNode):
       self.sim = ExternalModel.plugins.returnPlugin("ExternalModel",paramInput.parameterValues['subType'],self)
     else:
       self.raiseAnError(IOError,'"ModuleToLoad" attribute or "subType" not provided for Model "ExternalModel" named "'+self.name+'"!')
+    if not self.pickled:
+      # check if there are variables and, in case, load them
+      for child in paramInput.subparts:
+        if child.getName() == 'variables':
+          self.raiseAWarning(DeprecationWarning ,'"variables" node inputted but has been depreciated!  Please list variables in the "inputs" and "outputs" nodes instead.  This Warning will result in an error in RAVEN 3.0!')
+          if len(child.parameterValues) > 0:
+            self.raiseAnError(IOError,'the block '+child.getName()+' named '+child.value+' should not have attributes!!!!!')
```

Comment:
shouldn't this be an input params check?

Reply:
It looks like it has just been copied from older code without realizing it.  Removing (since variables is already takes no subs).